### DISCUSSION TITLE
Server Error when I try to import code into an empty repository

### DISCUSSION THREAD

**Discussion by Nich-Brew** [community/community#120417] [Repositories]
Bug

I tried making an empty repository and selecting the option to "import code from another repository", but every time I do I get a 500 error. The only thing it tells me is that "Oops!!! Looks like something went wrong!" I've tried refreshing, making a different empty repository, clearing my cookies, and using a different browser and nothing seems to work. I was able to use this option a couple days ago with no issues and GitHub status says everything is working properly. I've made a few google searches as well, but I couldn't find anything talking about this issue. If anyone has any fixes, I would greatly appreciate it!

**Comment by PROXBOY** [comment]
Same problem

**Comment by carlosmendezkare** [comment]
Same problem.

**Comment by giobongio** [comment]
Same here, another thread says that this can be caused if the repo is private but I have a public repo.

**Comment by Wayrion** [comment]
+1 I'm trying to import a repository from a private GitHub org to another GitHub org, and it keeps failing. I've tried Username/Password and Username/Access Key for authentication but both of them seem to fail.

**Comment by ghost** [comment]
same [
![githuberror](https://github.com/community/community/assets/161864408/57445a6a-125b-42d8-8646-2ec0cb334a4f)
](url)

**Comment by erigglem** [comment]
If you click the "plus sign" icon and then click "Import repository"
![image](https://github.com/community/community/assets/25778308/08639373-1648-41eb-9eb1-0fb1de31d732)

...it will take you to a screen where you can enter the details of the repo to be imported as well as the details about the new repo you'll be creating/importing to.

When I used this page, I still got the server 500 error. 

But, after about 3-4 minutes I received an email that my import had finished. I can also verify that it's imported successfully, despite still seeing the error.
